.laminas-ci.json

```diff
@@ -1 +1,15 @@
-{}
+{
+    "additional_checks": [
+        {
+            "name": "Run Tests on 8.4",
```

Comment:
This shouldn't be necessary. Even `lowest` should be properly handled via `ignore_php_platform_requirements` as we added that newish composer feature: `--ignore-platform-req=php+` (which respects upper bounds)

Ref: https://github.com/composer/composer/pull/10318

Reply:
Oh, nvm: 

https://github.com/laminas/laminas-continuous-integration-action/issues/179

Maybe implement that first so we can leave this alone.

Reply:
👍 I'm just mucking around really - I didn't know how to run checks for PHP versions not listed in composer.json. Led me to find this issue: https://github.com/laminas/laminas-ci-matrix-action/pull/310